src/runtime_src/core/edge/drm/zocl/zocl_aie.c

```diff
@@ -131,9 +266,15 @@ zocl_create_aie(struct drm_zocl_dev *zdev, struct axlf *axlf)
 
 	zdev->aie->partition_id = req.partition_id;
 	zdev->aie->uid = req.uid;
+
+	/* Register AIE error call back function. */
+	rval = aie_register_error_notification(zdev->aie->aie_dev,
```

Comment:
Why the word wrap? Would it be easier to read if the code as all on one line.

Reply:
same as above